### ISSUE TITLE
Linux DNS not working when AdGuard Home is not running

### ISSUE THREAD

**Issue by smcclennon** [AdguardTeam/AdGuardHome#2213]
### Issue Details
I have made my network stop using Adguard Home (changed router DNS to Cloudflare, enabled router DHCP, disabled Adguard DHCP).

When **AdGuard Home is not running on my Fedora machine, DNS no longer works**. I can access websites via their IPv4 address, so internet is working.

When I run AdGuard Home, DNS begins to work again on my machine again.

The rest of the network works regardless of whether AdGuard Home is running on my machine as I previously mentioned.

* **Version of AdGuard Home server:**
  * 0.103.3

* **How did you setup DNS configuration:**
  * Router

* **If it's a router or IoT, please write device model:**
  * Linksys MR8300v1

* **Operating system and version:**
  * Fedora 32

### Expected Behavior

My  machine should be able to access github.com for example regardless of whether AdGuard Home is running or not.
### Actual Behavior

My machine can only access github.com when AdGuard Home is running

</details>

### Additional Information

I have tried disabling automatic DNS on my machine and manually entering `1.1.1.1`, same problem. DHCP and DNS is working perfectly fine on all other devices connected to the network.
- Protection is disabled in AdGuard Home
- DHCP is disabled in AdGuard Home

I installed AdGuard Home over two months ago (and had it continuously running until today, when I wanted to uninstall it), and I do remember the installation being pretty rough so it is possible that I have followed "extra" steps based on an issue or third-party instructions to get it working that I have forgotten about.

**Comment by ameshkov** [comment]
Tbh I don't know how exactly have you configured your devices to use AGH.

You could've done it in your router or you could've changed DNS in `/etc/resolv.conf`

**Comment by smcclennon** [comment]
Did some more troubleshooting and figured it out.

For some reason, during AdGuard installation I'm pretty sure I disabled `systemd-networkd` and `systemd-resolved` (cannot remember why)

To fix my issue, all I did was:

1. Create symbolic link (if it doesn't already exist)
```
sudo ln -sf /run/systemd-resolve/resolv.conf /etc/resolv.conf
```

2. Start and enable `systemd-networkd`
```
sudo systemctl enable systemd-networkd
sudo systemctl start systemd-networkd
```

3. Start and enable `systemd-resolved`
```
sudo systemctl enable systemd-resolved
sudo systemctl start systemd-resolved
````
